codeforamerica/vita-min
# TBE-84 Bumped dependencies

## [TBE-84](https://codeforamerica.atlassian.net/browse/TBE-84)
## Is PM acceptance required?
- No - merge after code review approval
## What was done?
- No Functional changes
- Updated `yarn` dependencies
## How to test?
- Make sure all specs pass
- Test on Heroku

[TBE-84]: https://codeforamerica.atlassian.net/browse/TBE-84?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
